### PR TITLE
build: add FHIR load test package

### PR DESCRIPTION
**Pull request by leite08**
Ref. metriport/metriport-internal#746

### Dependencies

none

### Description

Introduces FHIR load test with Artillery.

This does not closes #746, it sets the base for the remaining tests.

### Release Plan

- it will lead to a deploy bc of a small change on `api/app/src/routes/util.ts` (log only)

### CODE REVIEW

File: fhir-test/src/example.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const functionFromProcessor = async (userContext, events, done) => {
+  userContext.vars.requestBodyFromProcessor = "something";
+  // continue with executing the scenario:
+  return done();
+};
+
+// https://www.artillery.io/docs/guides/guides/http-reference#beforerequest-hooks
+function setJSONBody(requestParams, context, ee, next) {
+  // ...
```
Comment (Goncharo) [inline fhir-test/src/example.js:9]: lefttover?

File: fhir-test/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+$ DEBUG=http:response artillery run --dotenv .env src/healthcheck-medplum.yml
+```
+
+To store the result of the tests on a file:
+
+```shell
+$ artillery run --dotenv .env --output test-run-report.json src/healthcheck-medplum.yml
+```
+
+And to generate a report in HTML
```
Comment (Goncharo) [inline fhir-test/README.md:51]: hyper nit: `in HTML:`

File: fhir-test/src/example.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+config:
+  target: "https://youserver.com"
```
Comment (Goncharo) [inline fhir-test/src/example.yml:2]: nit: `yourserver`

**Review by Goncharo — APPROVED** [review]
LGTM - few nit comments

File: fhir-test/src/example.js (hunk 1 above)
Comment (leite08) [reply fhir-test/src/example.js:9]: no, just a placeholder for code whoever takes the example as a reference,

**Pull request merged**